tests/hw8_search_test.py

```diff
@@ -0,0 +1,90 @@
+import time
+import unittest
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service
+from selenium.webdriver.common.by import By
+from webdriver_manager.chrome import ChromeDriverManager
+
+
+class SearchTestCase(unittest.TestCase):
+
+    def setUp(self) -> None:
+        browser = webdriver.Chrome(service=Service(executable_path=ChromeDriverManager().install()))
+        browser.get('http://hrm-online.portnov.com/')
+        self.browser = browser
+
+
+    def test_search(self, search_result=None):
+        browser = self.browser
+        browser.find_element(By.CSS_SELECTOR, "#txtUsername").send_keys("admin")
+        browser.find_element(By.XPATH, "//input[@id='txtPassword']").send_keys("password")
+        browser.find_element(By.CSS_SELECTOR, '.button').click()
+        self.assertIn("/pim/viewEmployeeList", browser.current_url)
+
+        welcome_message = browser.find_element(By.ID, "welcome").text
+        self.assertEqual('Welcome Admin', welcome_message)
+
+        # Assertion that QA Engineer role exists within the list
+        content = browser.find_element(By.XPATH, "//option[text()='QA Manager']").text
+        self.assertEqual("QA Manager", content)
+
+        # Assertion that Part Time option is present
+        employment_status = browser.find_element(By.XPATH, "//option[text()='Part Time']").text
+        self.assertEqual("Part Time", employment_status)
+
+        # Selecting QA Manager and Part Time filters
+        browser.find_element(By.XPATH, "//option[text()='QA Manager']").click()
+        browser.find_element(By.XPATH, "//option[text()='Part Time']").click()
+
+        browser.find_element(By.CSS_SELECTOR, "#searchBtn").click()
+
+        time.sleep(2)
+
+        # Adding assertion for the result list for QA Manager with a Part Time Employment Status using the for loop.
+        job_title = "QA Manager"
+        job_type = "Part Time"
+        search_result_list = browser.find_elements(By.XPATH, "//td[text()='QA Manager']")
+        search_job_type = browser.find_elements(By.XPATH, "//td[text()='Part Time']")
+
+        for qa_manager in search_result_list:
+            print(qa_manager.text)
+            if job_title == qa_manager.text:
+                self.assertTrue(job_title, qa_manager)
+            else:
+                self.assertFalse(job_title, qa_manager)
```

Comment:
VERY DANGEROUS CODE
You are creating a FALSE positive test. In other words, your test always passes, even when it should and must fail